server/views/components/find-us/index.njk

```diff
@@ -0,0 +1,11 @@
+<div class="find-us">
+  {% icon 'other/location' %}
+  <p class="find-us__address" itemprop="address" itemscope itemtype="https://schema.org/PostalAddress">
+    <span class="find-us__street" itemprop="streetAddress">183 Euston Road,</span>
+    <span class="find-us__locality" itemprop="addressLocality">London</span> 
+    <span class="find-us__postal-code" itemprop="postalCode">NW1 2BE</span>
+  </p>
+  <a class="find-us__link" href="https://www.google.com/maps/dir//nw12be/">
+    <img class="find-us__map" src="https://maps.googleapis.com/maps/api/staticmap?size=288x288&scale=2&markers=size:mid%7Ccolor:0xff0000%7Clabel:%7Cnw12be&key=AIzaSyCVivnLAgwcYsOqwitxK_Lb6LxSr37v4tc" alt="Directions to Wellcome Collection">
```

Comment:
Are we going to add a more interactive version? Or potentially a link saying "Directions" Which could like off to the big GMaps with us prefilled as the first address (most small / embedded maps are crap when you add interactivity, which is why I like just the image)?